contrib/buildahimage/centos7/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# stable/Dockerfile
+#
+# Build a Buildah container image from the latest
+# stable version of Buildah on the Fedoras Updates System.
```

Comment:
Being Centos, it doesn't pull the image from the Fedoras Updates System does it?